apps/web/tests/variants.spec.ts

```diff
@@ -148,7 +148,7 @@ test('shold not allow adding variants for delay step', async ({ page }) => {
   expect(page.getByTestId('add-variant-action')).toHaveCount(0);
 });
 
-test('should show step actions with no variants', async ({ page }) => {
+test.skip('should show step actions with no variants', async ({ page }) => {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please add code comment with reason for skipping‏